layouts/partials/tw/donate-prefill-lg.html

```diff
@@ -0,0 +1,184 @@
+{{ $hed := .hed | default "Become a Member" }}
+{{ $cta := .cta | default "Donate Now" }}
+
+{{ $btnClass := `outline-none min-w-[180px] inline-block rounded py-3 px-6 text-white font-sans font-black text-lg tracking-wide ring ring-transparent` }}
+{{ $activeClass := `bg-red-6 text-white hover:bg-red-5 focus:ring-red-7 active:ring-red-9` }}
+{{ $inactiveClass := `bg-s-2 text-s-9 hover:bg-s-3 focus:ring-s-7 active:ring-s-9` }}
+{{ $radioClass  :=  `rounded-full border-s-1 appearance-none mr-4 order-first w-6 h-6 checked:bg-black shadow-[0_0_0_3px_rgba(255,255,255,1)]`}}
```

Comment:
That shadow style is probably useful enough to add to the tailwind.config.js file, although that can wait until this is closer to merging.

Reply:
Are you referring to the radio input shadow?

Reply:
I've added this to tasks for building out the evergreen donate form.